acp/ideas_module.php

```diff
@@ -42,12 +45,18 @@ class ideas_module
 	/** @var string */
 	public $u_action;
 
+	/**
+	 * Constructor
+	 *
+	 * @access public
+	 */
 	public function __construct()
 	{
-		global $config, $db, $phpbb_log, $request, $template, $user, $phpbb_root_path, $phpEx;
+		global $config, $db, $phpbb_container, $phpbb_log, $request, $template, $user, $phpbb_root_path, $phpEx;
 
 		$this->config = $config;
 		$this->db = $db;
+		$this->language = $phpbb_container->get('language');;
```

Comment:
Why do we inject it via container? If so, we can do it for all other injections, no need to inject them independently.